packages/ui/src/council/components/election/pastElection/PastElectionStats.tsx

```diff
@@ -42,5 +54,13 @@ export const PastElectionStats = ({
         denominator={totalVotes + ' votes'}
       />
     </StatsBlock>
+    <StatisticItem title="Revealed Stake">
+      <TextSmall>
+        <FractionValue
+          numerator={<ValueInJoys size="xs">{formatJoyValue(totalRevealedVoteStake, { precision: 2 })}</ValueInJoys>}
+          denominator={<ValueInJoys size="xs">{formatJoyValue(totalVoteStake, { precision: 2 })}</ValueInJoys>}
+        />
+      </TextSmall>
+    </StatisticItem>
```

Comment:
Would be great if you could make it like this:
![image](https://github.com/user-attachments/assets/b947ff4b-2928-4054-abf5-21ccd9733554)

You just need to slightly modify `Figure` styles in `StatisticBar` and then replace this code with
```suggestion
      <StatisticBar
        title="Revealed stake"
        value={totalRevealedVoteStake.divn(1e6).toNumber() / totalVoteStake.divn(1e6).toNumber()}
        numerator={<ValueInMJoys as={'span'} size="xs">{formatJoyValue(totalRevealedVoteStake.divn(1e6), { precision: 2 })}</ValueInMJoys>}
        denominator={<ValueInMJoys as={'span'} size="xs">{formatJoyValue(totalVoteStake.divn(1e6), { precision: 2 })}</ValueInMJoys>}
      />
```